From 8185e79cbc6ce3e53006861c8edc0c7b26dc06d8 Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date: Tue, 30 Dec 2014 22:18:20 +0200
Subject: [PATCH] Make the call to configure_package_config_file work with CMake 3.1.0.

configure_package_config_file()'s PATH_VARS option expects several
separate variable names instead of one list with all the names. Passing
the values in the expected format fixes the following error:

CMake Error at /usr/local/share/cmake/Modules/CMakePackageConfigHelpers.cmake:231 (message):
  Unknown keywords given to CONFIGURE_PACKAGE_CONFIG_FILE():
  "INCLUDE_DIR;SDK_DIR"
Call Stack (most recent call first):
  CMakeLists.txt:226 (configure_package_config_file)
  CMakeLists.txt:279 (kadu_configure_package_config_file)
---
 CMakeLists.txt |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ef624a9..315bb5e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -221,11 +221,10 @@ macro (kadu_configure_package_config_file _in _out _mode)
 	elseif ("${_mode}" STREQUAL INSTALL_TREE)
 		set (INCLUDE_DIR "${INSTALL_INCLUDE_DIR}")
 		set (SDK_DIR "${INSTALL_SDK_DIR}")
-		list (APPEND path_vars INCLUDE_DIR SDK_DIR)
 
 		configure_package_config_file ("${_in}" "${_out}"
 			INSTALL_DESTINATION "${INSTALL_CMAKE_DIR}"
-			PATH_VARS ${path_vars}
+			PATH_VARS INCLUDE_DIR SDK_DIR
 			NO_SET_AND_CHECK_MACRO NO_CHECK_REQUIRED_COMPONENTS_MACRO
 		)
 	endif ()
-- 
1.7.1

